sabnzbd/assembler.py

```diff
@@ -326,14 +326,14 @@ def check_encrypted_and_unwanted_files(nzo, filepath):
                     if is_cloaked(filepath, zf.namelist()):
                         nzo.encrypted = 1
                         encrypted = True
-                    elif not sabnzbd.HAVE_CRYPTOGRAPHY and not passwords:
```

Comment:
I removed this. I dont understand why this is needed. I dont understand the comment it makes no sense to me.